### PR TITLE
Add assets

### PR DESCRIPTION
**Pull request by thangtd96**

### CODE REVIEW

**Comment by thangtd96** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -38,6 +38,7 @@ def self.find_first_by_auth_conditions warden_conditions
   end
 
   def send_devise_notification(notification, *args)
-    devise_mailer.send(notification, self, *args).deliver_later
+    devise_mailer.send(notification, self, *args).deliver_later wait: 10.seconds
```
Comment (tuyenvd1177) [inline app/models/user.rb]: cho vào settings

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -8,6 +8,14 @@
 
         <%= devise_error_messages! %>
 
+         <%= f.label :username %>
```
Comment (tuyenvd1177) [inline app/views/devise/registrations/edit.html.erb]: indent này

**Comment by tuyenvd1177** [comment]
commented

**Comment by thangtd96** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -10,6 +10,9 @@
   <body>
     <div class="arlina">
       <%= render "shared/header" %>
+      <% flash.each do |message_type, message| %>
+        <%= content_tag(:div, message, class: "alert alert-#{message_type}") %>
```
Comment (tuyenvd1177) [inline app/views/layouts/application.html.erb]: remove ()

**Comment by tuyenvd1177** [comment]
commented

**Comment by thangtd96** [comment]
ready

**Comment by tuyenvd1177** [comment]
merged

**Pull request merged**